libraries/Microsoft.Bot.Connector/Authentication/EmulatorValidation.cs

```diff
@@ -70,6 +70,18 @@ public static bool IsTokenFromEmulator(string authHeader)
                 return false;
             }
 
+            var tid = token.Claims.FirstOrDefault(claim => claim.Type == "tid")?.Value != null ? '/' + token.Claims.FirstOrDefault(claim => claim.Type == "tid")?.Value + '/' : string.Empty;
```

Comment:
We could create a variable for `token.Claims.FirstOrDefault(claim => claim.Type == "tid")?.Value`, since its being used twice.
And i think it will be better to do the concatenation of the `/` where its used `var newIssuer = "https://sts.windows.net" + tid;`

Reply:
That "/" is useful to apply the start and end of the tenant id and be more exact with the search